src/components/profile/AskerConsultingTypeData.tsx

```diff
@@ -33,7 +33,11 @@ export const AskerConsultingTypeData = () => {
 										text={
 											topics.find(
 												(topic) =>
-													topic.id === resort.topic
+													userData.sessions.find(
+														(session) =>
+															session.agencyId ===
+															resort.agency.id
+													).topic.id === topic.id
```

Comment:
Is this working if no session was found? Maybe adding a ? to `...)?.topic.id` would make it more secure...